src/OpenTelemetry/.publicApi/net461/PublicAPI.Unshipped.txt

```diff
@@ -63,7 +63,8 @@ OpenTelemetry.Metrics.MeterProviderExtensions
 OpenTelemetry.Metrics.Metric
 OpenTelemetry.Metrics.Metric.Description.get -> string
 OpenTelemetry.Metrics.Metric.GetMetricPoints() -> OpenTelemetry.Metrics.MetricPointsAccessor
-OpenTelemetry.Metrics.Metric.Meter.get -> System.Diagnostics.Metrics.Meter
+OpenTelemetry.Metrics.Metric.MeterName.get -> string
+OpenTelemetry.Metrics.Metric.MeterVersion.get -> string
```

Comment:
What do folks think about this change? In part I was thinking since Meter can be disposed, maybe it's safer to not keep a handle on it from the metric. Another option would be to create a new struct to encapsulate these fields.

```csharp
namespace OpenTelemetry.Metrics;

public struct InstrumentationScope
{
    string Name { get; }
    string Version { get; }
    // Eventually
    string SchemaUrl { get; }
}
```

Reply:
This is a good improvement. Don't know if we need to introduce new Struct for this.

Reply:
Need to add this to changelog as this is breaking any existing exporters.